### ISSUE TITLE
Create a demo

### ISSUE THREAD

**Issue by ALEZ-DEV** [SorenHolstHansen/leptos_toaster#21]
Create a demo is the best to convince people to use your library, so I think that's a good idea and I think this is relatively quick to do

**Comment by SorenHolstHansen** [comment]
Yeah, I agree, I am pretty busy at the moment however, so can't promise anything
